app/connectors/DesConnector.scala

```diff
@@ -44,6 +44,11 @@ trait DesConnector {
                                                headerCarrier: HeaderCarrier,
                                                ec: ExecutionContext,
                                                request: RequestHeader): Future[Either[HttpException, PsaSubscription]]
+
+  def ceasePSA(psaToBeCeased: PsaToBeRemovedFromScheme)(implicit
```

Comment:
can we remove this to removePSA? (to go inline with the terminology in the frontend, starting with the `remove` link)